sdk/src/test/tests/AsyncTest.ls

```diff
@@ -0,0 +1,40 @@
+/*
+===========================================================================
+Loom SDK
+Copyright 2011, 2012, 2013 
+The Game Engine Company, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+===========================================================================
+*/
+
+package tests {
+
+    import unittest.Assert;
+    import unittest.TestComplete;
+    
+    public class AsyncTest {
```

Comment:
See my above comment "As for the async tests... That was my original intention, but it seems that for some reason the loom tests don't have access to any packages other that system and unittest packages. So no timers and no httprequests. I suspect this is not intended"